sfputil/main.py

```diff
@@ -289,6 +290,30 @@ def is_sfp_present(port_name):
 
     return bool(presence)
 
+
+# Determine whether it is a RJ45 port
+def is_rj45_port_from_db(port_name, db):
+    intf_type = db.get(db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(port_name), 'type')
+    return intf_type == RJ45_PORT_TYPE
+
+
+def is_rj45_port_from_api(port_name):
```

Comment:
@keboliu can put comments in the code why we need two different functions for getting the port type? Also when should one use each of them?

Reply:
@prgeor please check the latest commits for the comments on the usage of the functions.

Reply:
@keboliu as we discussed, lets have sfptuil always fetch the values by reading the eeprom and sfpshow utility use the DB to read the values. Hope this will be scoped and fixed in future PR.